XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Hands/HandData.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using XRTK.Definitions.Utilities;
+
+namespace XRTK.Providers.Controllers.Hands
+{
+    /// <summary>
+    /// Snapshot of hand data.
+    /// </summary>
+    [Serializable]
+    public class HandData
+    {
+        /// <summary>
+        /// Timestamp of hand data, as FileTime, e.g. DateTime.UtcNow.ToFileTime().
+        /// </summary>
+        public long TimeStamp { get; set; } = 0;
+
+        /// <summary>
+        /// Is the hand currently being tracked by the system?
+        /// </summary>
+        public bool IsTracked { get; set; } = false;
+
+        /// <summary>
+        /// Pose information for each hand joint.
+        /// </summary>
+        public MixedRealityPose[] Joints { get; } = new MixedRealityPose[BaseHandController.JointCount];
```

Comment:
should we initialize this with an empty array?

How do we get a reference to the `BaseHandController.JointCount`? Is this different per controller?